dftimewolf/lib/recipes/manager.py

```diff
@@ -13,10 +13,18 @@
 
 
 class RecipesManager(object):
-  """Recipes manager."""
+  """Recipes manager.
+
+  Attribs:
+    enable_overwrite: Whether an exception is raised if an existing module is
```

Comment:
I opt to change this to a class constant named `ALLOW_RECIPE_OVERRIDE` and please add some tests